src/netex-convertor/testdata/operatorData.ts

```diff
@@ -0,0 +1,12 @@
+const operatorData = {
```

Comment:
Does this one need its own testdata file or could it live in test-data.ts?

Reply:
Have now moved operatorData into test-data.